packages/fx-core/src/plugins/resource/keyvault/constants.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export class Constants {
+  static readonly provisionModuleTemplateFileName: string = "keyVaultProvision.template.bicep";
+  static readonly configModuleTemplateFileName: string = "keyVaultConfiguration.template.bicep";
+
+  static readonly SolutionPlugin = {
+    id: "solution",
+    configKeys: {
+      remoteTeamsAppId: "remoteTeamsAppId",
+    },
+  };
+
+  static readonly KeyVaultPlugin = {
+    pluginName: "fx-resource-key-vault",
+    displayName: "Key Vault Plugin",
+    shortName: "kv",
+  };
+
+  static readonly Stage = {
+    generateArmTemplates: "generate-arm-templates",
```

Comment:
Is there existing const to have the same stage value?

Reply:
Unfortunately, no...